pkgs/development/libraries/gd/default.nix

```diff
@@ -42,7 +43,7 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ autoconf automake pkg-config ];
 
   buildInputs = [ zlib fontconfig freetype ];
-  propagatedBuildInputs = [ libpng libjpeg libwebp libtiff libXpm ];
+  propagatedBuildInputs = [ libpng libjpeg libwebp libtiff libXpm libavif ];
```

Comment:
Does this need to be propagated? `libavif` is in `Requires.private` in `${gd.dev}/lib/pkgconfig/gdlib.pc` so our patched `pkg-config` should should not need it to be propagated.

Reply:
A derivation with `libavif` placed within `buildInputs` seems to work as well. What about the other inputs which have already been propagated?

Reply:
It appears they have been introduced in https://github.com/NixOS/nixpkgs/commit/8b2fc35987e62c079fd50980782495360039a9c1. Not really much of context.

Reply:
Done